core/src/main/scala/Released.scala

```diff
@@ -30,7 +30,7 @@ final case class Released(
   /* when was this release created */
   timestamp: Instant,
   /* reference id from github */
-  releaseId: Long,
+  releaseId: String,
```

Comment:
Why is this a `String` now? does Gitlab not represent it was a number?

Reply:
Unfortunately, it's not the case. Gitlab has no `releases`, just `tags`, and these have no numeric `ids` either. :disappointed: